inst/EpiModel2.rmd

```diff
@@ -56,13 +63,13 @@ target.stats <- c(30, 0)
 
 It is important to note, that the vertex attribute "group" now takes on a special meaning within the context of EpiModel modeling, separate from other attribute names. Users should be mindful when constructing networks in which "group" is a vertex attribute and note whether or not the interpretation of group aligns with that detailed above.
 
-Further, the group vertex attribute is restricted to levels of 1 and 2; other combinations of attribute levels - 0 and 1 or "male" and "female" - are not permitted here and will result in unexpected behavior in EpiModel. Further note that we are not limited to replicating bipartite networks and the target statistic for `nodematch("group")` can be any number from 0 to the total number of edges in the network.
+Further, the group vertex attribute is restricted to levels of 1 and 2; other combinations of attribute levels - say, 0 and 1 or "male" and "female" - are not permitted here and may result in EpiModel producing unintended results. Finally, we are not limited to replicating bipartite networks and the target statistic for `nodematch("group")` can be any number from 0 to the total number of edges in the network.
 
 The above restrictions on the group vertex attribute must be taken into account or EpiModel may behave in unpredictable or unexpected ways.
```

Comment:
again adding `` around group for consistency - see earlier comment on whether it should be "group"

```suggestion
The above restrictions on the `group` vertex attribute must be taken into account or EpiModel may behave in unexpected ways.
```